src/CSS/Header.css

```diff
@@ -5,4 +5,10 @@
 .header_selector {
   border: none !important;
   margin: 2px !important;
-}
\ No newline at end of file
+}
+
+.stickyClass {
```

Comment:
Please do not use camelCase in css class names.

Reply:
changed to snake case. thanks.